hack/fetch-official-cve-feed.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import sys
+import requests
+
+#print("\nToken:", sys.argv[1])
+
+url = 'https://api.github.com/search/issues?q=is:issue+CVSS:3+in:body+label:committee/\
+security-response+state:closed+repo:kubernetes/kubernetes'
```

Comment:
```suggestion
url = 'https://api.github.com/search/issues?q=is:issue+label:official-cve-feed+state:closed+repo:kubernetes/kubernetes'
```

Reply:
Can we update the URL as above? This is the most accurate filter for the CVE feed

Reply:
Updated